src/Microsoft.PowerShell.Commands.Utility/commands/utility/Get-Error.cs

```diff
@@ -15,6 +15,7 @@ namespace Microsoft.PowerShell.Commands
     [Cmdlet(VerbsCommon.Get, "Error",
         HelpUri = "https://docs.microsoft.com/powershell/module/microsoft.powershell.utility/get-error?view=powershell-7&WT.mc_id=ps-gethelp",
         DefaultParameterSetName = NewestParameterSetName)]
+    [OutputType("PSExtendedError")]
```

Comment:
```powershell
 (Get-Error).Gettype()

IsPublic IsSerial Name                                     BaseType
-------- -------- ----                                     --------
True     True     ErrorRecord                              System.Object
```

```powershell
 $a.psobject.TypeNames
PSExtendedError
PSExtendedError
PSExtendedError
System.Object
```
I wonder to see the difference.
Also for second output I'd expect to see `ErrorRecord` too, not only 3 (!) fake types.

Reply:
The multiple is due to everytime you `Get-Error()`, it inserts it.  I'll fix that in this PR.

Reply:
Interesting, I hadn't thought about that this would mutate the object. So then, after you called `Get-Error` once, `$error` would suddenly also show the same formatting, right? With no way to get the original formatting again other than manually mutating again? That's a little surprising.

Is there no way the format could just be defined for the `ErrorRecord` type, but as a `CustomControl` that is not used by default but only in certain contexts?

Reply:
It would probably be prudent to simply `Copy()` the PSObject wrapper and maintain the internals, so you only add the PSTypeName to a fresh instance.

Reply:
`Copy()` copies the refs so it doesn't help here as I'm still manipulating the original object.